nodejs/packages/layer/src/wrapper.ts

```diff
@@ -212,6 +226,44 @@ function createInstrumentations() {
   ];
 }
 
+function getPropagator(): TextMapPropagator {
+  if (
+    process.env.OTEL_PROPAGATORS == null ||
+    process.env.OTEL_PROPAGATORS.trim() === ''
+  ) {
+    return new CompositePropagator({
+      propagators: [
```

Comment:
Could you clarify why the default propagators will not match the ones listed here? https://github.com/open-telemetry/opentelemetry-lambda/blob/main/nodejs/packages/layer/scripts/otel-handler#L12

Also if the defaults are already set in otel-handler, is there any reason to also set it here? (I actually prefer that to be here instead)

Reply:
@pragmaticivan Because this is the **existing** behavior. 

Currently, propagators are taken from here `https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/metapackages/auto-configuration-propagators/src/utils.ts#L51` and hence I used the impl/logic there.

Reply:
But the defaults of `this project` are `tracecontext,baggage,xray` instead. This env variable will always be overwritten by that file, so the code will never hit that block

Reply:
- without this PR, default propagators (tracecontext,baggage,xray) are set by the `otel-handler` script
- with this PR, still default propagators (tracecontext,baggage,xray) are set by the `otel-handler` script

So, this keeps the existing behavior. No need to put the same default list into the newly introduced `getPropagator` method too. `otel-handler` script is responsible for setting the defaults.

Reply:
I don't understand the issue here. This PR keeps the existing behavior for propagators by default because it uses the existing impl code. Why it was OK before, and NOT OK now?

Reply:
I guess you are trying to replicate the `other package behavior here` when you might not need this code block. All good, though. I understand that this was the default from the other package, yet this is a block of code that will `never` be able to be reached `in this package`, which makes it `dead code` out of the gate

Reply:
No one can guarantee that this is dead code even though `otel-handler` is in place. 

Because users might put their own initializers and activate it through the `NODE_OPTIONS`. And in this case, their initializers will run before our wrapper. So, potentially, they can clear the `OTEL_PROPAGATORS` env var in their code.

Reply:
Makes sense, I guess I see the point of this code, but don't see the point of keeping that code in otel-handler 😂

Reply:
If you are still interested in removing that code block, you can send another PR for it and we can discuss. But the main point of this PR is **dropping** b3 and jaeger propagator dependencies **without** changing existing behavior.